src/status_im/contexts/keycard/pin/create/view.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im.contexts.keycard.pin.create.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.constants :as constants]
+            [utils.i18n :as i18n]))
+
+(defn view
+  [{:keys [on-complete]}]
+  (let [[pin set-pin]             (rn/use-state "")
+        [first-pin set-first-pin] (rn/use-state "")
+        [error? set-error]        (rn/use-state false)
+        [stage set-stage]         (rn/use-state :create)
+        pin-empty?                (string/blank? pin)
+        on-delete                 (rn/use-callback
+                                   (fn []
```

Comment:
Hi @flexsurfer, Thank you for your PR.

Maybe we should create functions separately for improving readability. wdyt?

```
on-delete (rn/use-callback #(on-delete-fn pin) [pin])
```

Reply:
thank you, yeah i thought about that , do you think it's better?

Reply:
i can do it in the next PRs